server/src/app/game_manager.py

```diff
@@ -0,0 +1,116 @@
+import string
+import random
+
+class GameManager:
+    def __init__(self, redis_client):
+        self.redis_client = redis_client
+        if not self.redis_client.exists("breach:games_count"):
+            self.redis_client.set("breach:games_count", 0)
+
+    def create_game(self):
+        new_game_id = int(self.redis_client.get("breach:games_count")) + 1
+        self.redis_client.set("breach:games_count", new_game_id)
+        board = self.generate_random_board(new_game_id)
+
+        # Create token for joining the game
+        token = self.generate_random_token()
+        self.redis_client.hset("breach:token", token, new_game_id)
+
+        self.redis_client.hset("breach:game_status", new_game_id, "started;waiting_player2")
+        game_state = {
+            'game_id': new_game_id,
+            'token': token,
+        }
+        game_state.update(board)
+        return game_state
+
+    def generate_random_board(self, game_id):
+        # generate and store board, with player positions
+        p1_x = 0
+        p1_y = 0
+        p2_x = 1
+        p2_y = 1
+
+        self.redis_client.hset("breach:player1_x_pos", game_id, p1_x)
+        self.redis_client.hset("breach:player1_y_pos", game_id, p1_y)
+        self.redis_client.hset("breach:player2_x_pos", game_id, p2_x)
+        self.redis_client.hset("breach:player2_y_pos", game_id, p2_y)
+        self.redis_client.hset("breach:player_turn", game_id, 1)
+
+        return {
+            'player1_pos': "{}, {}".format(p1_x, p1_y),
```

Comment:
How about returning pos as a tuple instead of a string? `'player1_pos': (p1_x, p1_y)`. Otherwise it requires on the javascript side some processing to get the coordinates back.

Reply:
I did the changes but had to modify index.html to have consistency on how it presents the coordinates after the user moves